README.md

```diff
@@ -187,6 +187,18 @@ var discovery = new DiscoveryV1({
   });
 ```
 
+### Setting the service URL
+
+You can set the service URL by calling the setServiceUrl() method.
+
+```javascript
```

Comment:
So this means an update to our API references, where we have the following: (?)
```javascript
var DiscoveryV1 = require('watson-developer-cloud/discovery/v1');

var discovery = new DiscoveryV1({
  version: '{version}',
  iam_apikey: '{apikey}',
  url: 'https://gateway-wdc.watsonplatform.net/discovery/api'
});
```

Reply:
Yes ... we can now offer the alternate means of setting the service URL for Node.